lib/manageiq/cross_repo/runner.rb

```diff
@@ -66,13 +78,16 @@ def generate_bundler_d
       override_path  = bundler_d_path.join("overrides.rb")
 
       if gem_repos.empty?
+        require "fileutils"
         FileUtils.rm_f override_path
       else
         content = gem_repos.map do |gem|
           # If there is a gemspec get the name of the gem from that instead of the repository
           gem_name = gem.path.glob("*.gemspec")&.first&.basename(".gemspec") || gem.repo
           "ensure_gem \"#{gem_name}\", :path => \"#{gem.path}\""
         end.join("\n")
+
+        require "fileutils"
```

Comment:
Might as well move this outside of the if/else clause since it's used it both parts.